apolloschurchapp/src/ui/LocationFeatureConnected/LocationFeature/index.js

```diff
@@ -19,27 +19,27 @@ export default function LocationFeature({
 
   return (
     <>
-      <MapView
-        initialRegion={{
-          latitude: lat,
-          longitude: long,
-          latitudeDelta: 0.05,
-          longitudeDelta: 0.05,
-        }}
-        style={{ aspectRatio: 2 / 1 }}
-      >
-        <Marker
-          latitude={lat}
-          longitude={long}
-          onPress={undefined}
-          opacityStyle={undefined}
-        />
-      </MapView>
+      {lat && (
```

Comment:
hey sorry there is a simple solution, I should've included that!

`{lat ? ( ... ) : null}`

Basically just make sure that when `lat` is falsey that you are returning `null` (or `undefined` or something that react-native won't choke on rendering). This is more just type-safety then anything else.

Reply:
Also it might make sense to check for both `lat` and `lon` - `{lat && long ? ( ... ) : null}` - as I think the map view will error if either of those are falsey.